Capfile

```diff
@@ -1,12 +1,12 @@
 # frozen_string_literal: true
 
 # Load DSL and Setup Up Stages
-require 'capistrano/setup'
+require "capistrano/setup"
```

Comment:
Use double-quotes by default instead of fussing about single vs. double depending on whether a string needs interpolation. Works well with Rails which generates files with double-quotes anyway.